content/02.introduction.md

```diff
@@ -67,6 +67,12 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
 *Presentation of COVID-19:* 
 Information is rapidly becoming available about the wide range of symptoms that can be associated with COVID-19 as well as the range of symptom severity, onset from exposure, and possible risk or protective factors...
 
+In a retrospective sample of COVID-19 patients in Wuhan observed during December 2019 and January 2020, [@doi:10.1016/S0140-6736(20)30566-3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment score, as well as high levels of d-dimer. 
+Mortality was also associated with other biomarkers including lactate dehydrogenase and cardiac troponin I. 
```

Comment:
```suggestion
Mortality was also associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I. 
```
Is there a mechanism mentioned via which these might be of interest or related to pathogenesis? For example, they seemed to have measured IL-6, but was that one not elevated? I'm also interested in the cardiac troponin I because it seems they used that to diagnose cardiac issues -- were those pre-existing or caused by COVID-19? That is apparently related to some concerns about chloroquinine (based on news coming out of Sweden #160 ) so it's very interesting if elevated levels of cardiac biomarkers are part of pathogenesis. Given that the stats in Table 1 were taken upon admission, I also worry about interpreting too broadly (because they might indicate more about who gets admitted for COVID-19 than how COVID-19 necessarily compares to baseline!)

Reply:
Those are really important concerns -- unfortunately I don't think I personally am qualified to answer them adequately.

Reply:
@dziakj1 I can take a closer look in the morning and let you know if I figure out anything about the biomarkers!

Reply:
Apparently a bunch of people are interested in cardiac troponin.   Would you like me to look into it some more?
https://www.sciencedirect.com/science/article/pii/S0033062020300554
https://www.acc.org/latest-in-cardiology/articles/2020/03/18/15/25/troponin-and-bnp-use-in-covid19
https://www.ahajournals.org/doi/abs/10.1161/CIRCULATIONAHA.120.047008
https://www.medpagetoday.com/infectiousdisease/covid19/85605

Reply:
OK I updated #143 with my concerns about these conclusions. Mostly I am very skeptical of their stats, and it's not clear what exactly table 3 is comparing. If you are interested in making a new issue about Cardiac Troponin, it seems like a very interesting avenue to explore!

Reply:
> I think these are really important validity concerns -- I am very sorry for missing them. Now that you point them out, I suppose that you shouldn't include this paper in the introduction without some mention of the limitations, but it is up to you.

@dziakj1 it's definitely an important study and I think most of the references are going to have some number of issues because of the rate they're coming out! I don't think you overstated its findings. If you agree about the multiple testing issue, what do you think of the updated suggestion I made above? Sorry for being so slow to get back to you on this today!